### ISSUE TITLE
Omon.tos: Getting the right standard name

### ISSUE THREAD

**Issue by martinjuckes** [cmip6dr/cmip7_forward_look#67]
The variable `tos` has CF Standard Name `sea_surface_temperature` which is defined as being the "near-surface temperature" which is distinct from interface temperature:

>  It is the temperature of sea water near the surface (including the part under sea-ice, if any). More specific terms, namely sea_surface_skin_temperature, sea_surface_subskin_temperature, and surface_termperature are available for the skin, subskin, and interface temperature. 

In [Griffies et al. 2016](https://gmd.copernicus.org/articles/9/3231/2016/gmd-9-3231-2016.pdf), on the other hand, `tos` is defined as:

> SST (tos) is the interface temperature at the upper boundary of the ocean

@durack1 , @taylor13 : I've just noticed this contradiction between the description in the OMIP GMD paper and the CF metadata.

**Comment by taylor13** [comment]
We should clean this up.  In many models these two temperatures are the same, I think, so even though their definitions are different, the quantity reported is unaffected.  I doubt if this is true of all models, however.

**Comment by durack1** [comment]
In my experience, the `tos` variable is the same as the top layer of the `thetao` variable (ocean realm, ocean model), which is most often a 0 to ~10 metre averaged quantity. This is different for satellite obs (also for `sos`) which most often report the ocean "skin" value, top ~cm. From memory this was cleaned up for salinity many moons ago (CF standard names), might have to revisit `tos` in the same light

**Comment by durack1** [comment]
Yeah so `sea_surface_salinity` is defined in part as "Sea surface salinity is the salt content of sea water close to the sea surface, often on the Practical Salinity Scale of 1978. However, the unqualified term 'salinity' is generic and does not necessarily imply any particular method of calculation. The units of salinity are dimensionless and the units attribute should normally be given as 1e-3 or 0.001 i.e. parts per thousand. Sea surface salinity is often abbreviated as "SSS"."

**Comment by durack1** [comment]
I think we're good looking at `sea_surface_temperature` "Sea surface temperature is usually abbreviated as "SST". It is the temperature of sea water near the surface (including the part under sea-ice, if any). More specific terms, namely sea_surface_skin_temperature, sea_surface_subskin_temperature, and surface_temperature are available for the skin, subskin, and interface temperature. respectively. For the temperature of sea water at a particular depth or layer, a data variable of sea_water_temperature with a vertical coordinate axis should be used."

**Comment by taylor13** [comment]
What I don't know is if any model distinguishes between its skin temperature (used, for example, in its calculation of upwelling longwave radiation at the surface) and the bulk temperature of the ocean surface layer (determined by the balance of energy entering or leaving the upper-most grid cell). Another possibility is that a model will include some sub-model of the the temperature profile near the surface, so that the "sea_surface_temperature" is not simply set to the bulk temperature of the upper-most ocean layer, but is determined in a more complicated way.  In either case, if a model doesn't simply set its radiating temperature to its near-surface temperature, then which number do we want?  I'd vote for ``sea_surface_temperature``, not ``sea_surface_skin_temperature``.  If Steve Griffies agrees, then he should perhaps issue a clarification regarding the [Griffies et al. 2016](https://gmd.copernicus.org/articles/9/3231/2016/gmd-9-3231-2016.pdf) document.

**Comment by martinjuckes** [comment]
Hi Karl, Paul : I've had a quick search in ES-DOC, and also checked some literature -- no sign of any modelling of skin temperature in CMIP6, though high resolution NWP models are starting to take it into account.

Some of the satellite microwave data is also bulk SST, even though the instrument is responding to skin temperature, because the published data is calibrated against in situ measurements of bulk data (Rayner et al : https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2002JD002670 ) .. so this is probably just an issue of cleaning up wording to avoid confusion among users.
